test/com/wsscode/pathom3/connect/planner_test.cljc

```diff
@@ -2205,15 +2338,15 @@
                                       ::pco/input   [:b]
                                       ::pco/output  [:c]}]
               ::eql/query           [{:a [:c]}]})
-           {::pcp/nodes                 {2 {::pco/op-name        'dyn
```

Comment:
Node number changes I would personally consider implementation details?

Reply:
Yes, no worries about that